### DISCUSSION TITLE
Downloading files should be still possible if editing is disabled.

### DISCUSSION THREAD

**Discussion by nickluger** [directus/directus#5105] [Feature Requests]
The title says it. Downloading a file doesn't mean modifying it - it's only read access. There should be the download icon here, but it's only shown, if editing is enabled. Shouldn't accessing still be allowed, if editing is disabled?
![Bildschirmfoto 2021-04-16 um 15 45 51](https://user-images.githubusercontent.com/32647249/115033537-d5a6e880-9eca-11eb-8231-7640d9e65ca5.png)

### Project details
Directus Version: 9.0.0-rc.54
Environment: production
OS: Linux 4.14.225-169.362.amzn2.x86_64
Node: 14.16.0

**Comment by rijkvanzanten** [comment]
Makes sense to me! Wanna open a PR for that? 🙂

**Reply by nickluger** [reply]
I was into React the last years, don't know Vue yet. Need to learn that (at least a bit), before I'm able to contribute. 😅  Though this might be a good starter?

**Reply by rijkvanzanten** [reply]
Totally! This should be a very easy tweak even if you're not at all familiar with Vue 🙂 I'll give you a little hint to start, it's in here

https://github.com/directus/directus/blob/1f4838dc824f1865d2b53fb75e4dce402b205e4c/app/src/interfaces/files/files.vue#L28-L43

**Reply by nickluger** [reply]
For ppl finding this > https://github.com/directus/directus/pull/6025
